content/tutorials/marketplace-swap-and-stake.md

```diff
@@ -1,38 +1,73 @@
 ---
 title: Swap and/or Stake Tokens
-description: 
+description:
 ---
 
 ## Swap OCEAN Tokens against datatokens
 
-1. Search for the desired asset published on the [Ocean Marketplace](https://market.oceanprotocol.com/).
+### Finding an asset
 
-2. Select **Trade** option and enter the amount of Ocean tokens you want to swap. The expected amount that the account will receive will be shown with the swap fees information.
+Search for the desired asset published on the <a href="https://v4.market.oceanprotocol.com/" target="_blank">Ocean Marketplace</a>.
 
-   ![swap part-1](images/marketplace/Swap-1.png 'Select trade')
+### Step 1: Understanding the risks
```

Comment:
I would rename this step to "Agree with the risks associated".. Also with a description:
"Please read carefully the risks associated with this operation and agree to move forward"